blocks/subscriptions-block/intl.json

```diff
@@ -7179,6 +7466,7 @@
 		"bn": "every %{billingCount} day(s) for %{durationCount} week(s)",
 		"bo": "every %{billingCount} day(s) for %{durationCount} week(s)",
 		"de": "every %{billingCount} day(s) for %{durationCount} week(s)",
+		"el": "every %{billingCount} day(s) for %{durationCount} week(s)",
```

Comment:
Is there any reason why some of the scripts in this file doesn't have translated. Just showing same script like english.

Reply:
The translations haven't been ordered yet, but we need a string for each language key or the Fusion translate functions will fail.